BUILDING.md

```diff
@@ -557,6 +557,23 @@ export CXX="ccache c++"        # add to ~/.zshrc or other shell config file
 export LDFLAGS="-fuse-ld=mold" # add to ~/.zshrc or other shell config file
 ```
 
+On Windows:
+
+Tips: follow <https://github.com/ccache/ccache/wiki/MS-Visual-Studio>, and you
+should notice that obj file will be bigger the normal one.
+
+First, install ccache, assume ccache install to c:\ccache, copy
+c:\ccache\ccache.exe to c:\ccache\cl.exe
```

Comment:
> Change to `CL.exe`, is less error prone to confuse with 1.

I agree:)

Reply:
> Also, during the tests, I got some errors because I installed the `ccache` via chocolatey and I copy the shim file instead of the original file.
> 
> I think we should not include the steps to install the `ccache`, just ask the user to install and ensure they have the `CL.exe` file.

Yes, these instructions will make doc here mess, on another side,
As chocolatey is welcome these day, I’m a little worried most of the user will do these step wrong.